src/components/ArticleList.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { getAllArticles } from "../api/api";
+import Article from "./Article";
+
+function ArticleList({ topic }) {
```

Comment:
It would be great to see a loading pattern here. Your api will spin down after 15 mins of inactivity and take some time to spin back up (free tier, so not so great performance). While that is spinning up a user would ideally want to know things are working.